### PR TITLE
[SDK-165]: Removed login button helper functions from Example project

### PR DESCRIPTION
**Pull request by echarrod**
Helper functions are now defunct.
Some modifications of the tests were required to accommodate this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,19 +91,17 @@ import "yoti_python_sdk"
 
 After creating your application on the [Yoti Dashboard](https://www.yoti.com/dashboard/), you need to download the `.PEM` key and save it *outside* the repo (keep it private).
 
-The following variables are then required for the SDK to work:
+The following variables are then required for the SDK to work, these are found under the different tabs of your Yoti application's settings page ([Yoti Dashboard](https://www.yoti.com/dashboard/)):
```
Comment (vassyz) [inline README.md]: to work. These are

File: README.md [hunk 2]
```diff
@@ -91,19 +91,17 @@ import "yoti_python_sdk"
...
 
-* `YOTI_CLIENT_SDK_ID` - is the SDK identifier generated by Yoti Dashboard in the Key tab when you create your app. Note this is not your Application Identifier which is needed by your client-side code
-* `YOTI_KEY_FILE_PATH` - is the path to the application pem file. It can be downloaded only once from the Keys tab in your Yoti Dashboard. (e.g. /home/user/.ssh/access-security.pem)
+* **`YOTI_APPLICATION_ID`** - This is used to configure the [Yoti Login Button](https://www.yoti.com/developers/documentation/#2-front-end-integration).
+* **`YOTI_CLIENT_SDK_ID`** - is the SDK identifier generated by Yoti Dashboard in the Key tab when you create your app. Note this is not your Application Identifier which is needed by your client-side code
```
Comment (vassyz) [inline README.md]: Uppercase first letter and fullstop throughout this list.

Comment (vassyz) [reply README.md]: I think this needs to be corrected in three Readme files.

Comment (echarrod) [reply README.md]: Do you mean we should be changing the first letter to uppercase, or shouldn't be? I will also start them with the same word for consistency (`this` or `is`)

Comment (vassyz) [reply README.md]: Doesn't matter, as long as it's consistent. Uppercase letter requires fullstop, lowercase doesn't.

**Pull request merged**